### ISSUE TITLE
AttributeError: 'tuple' object has no attribute 'copy'

### ISSUE THREAD

**Issue by bitcometz** [theislab/cpa#4]
Hello,

I had download the pretrained model:

```
146722 Jun 20 04:51 attr.pkl
89559 Jun 20 04:51 history.csv
39920131 Apr 12  2021 model_params.pt
39022 Jun 20 04:51 var_names.csv
```

And I run
```python
adata = sc.read('/path/GSM_new.h5ad')
cpa.CPA.setup_anndata(adata,
                      drug_key='condition',
                      dose_key='dose_val',
                      categorical_covariate_keys=['cell_type'],
                      control_key='control',
                      combinatorial=True,
                     )

model = cpa.CPA.load('/Path/GSM/test/', adata, use_gpu=True)
```

But I got errors:
```
   1551 # copy state_dict so _load_from_state_dict can modify it
   1552 metadata = getattr(state_dict, '_metadata', None)
-> 1553 state_dict = state_dict.copy()
   1554 if metadata is not None:
   1555     # mypy isn't aware that "_metadata" exists in state_dict
   1556     state_dict._metadata = metadata  # type: ignore[attr-defined]

AttributeError: 'tuple' object has no attribute 'copy'
```

It is ok when I load my own pretrained models.

Could you help with this problem ?

Thanks !!!

**Comment by Naghipourfar** [comment]
Hi @bitcometz,

Just released a v0.3.0 version. Should be Ok now.

**Comment by bitcometz** [comment]
Thanks ！！！
